packages/stream/aptos/constants.ts

```diff
@@ -7,3 +7,26 @@ export const APTOS_PROGRAM_IDS: Record<ICluster, string> = {
   [ICluster.Testnet]: "0xc6737de143d91b2f99a7e490d4f8348fdfa3bdd1eb8737a27d0455f8a3625688",
   [ICluster.Local]: "0x9009d93d52576bf9ac6dc6cf10b870610bcb316342fef6eff80662fbbfce51b0",
 };
+
+export const APTOS_MODULE_ERROR_MAP: { [key: string]: { [key: number]: string } } = {
+  protocol: {
+    1: "ECONTRACT_NOT_INIT",
```

Comment:
Love it! Let's just put these value strings in some kind of ENUM, since each chain is mapping to those. We can also export than enum so FE can use it